src/materials/GeneralizedExternalDrivingForce.C

```diff
@@ -0,0 +1,128 @@
+//* This file is part of the RACCOON application
+//* being developed at Dolbow lab at Duke University
+//* http://dolbow.pratt.duke.edu
+
+#include "GeneralizedExternalDrivingForce.h"
+#include "Function.h"
+// #include "RankTwoTensorImplementation.h"
+// #include "ADRankTwoTensorForward.h"
+
+registerADMooseObject("raccoonApp", GeneralizedExternalDrivingForce);
+
+InputParameters
+GeneralizedExternalDrivingForce::validParams()
+{
+  InputParameters params = ADMaterial::validParams();
+  params.addClassDescription("computes the Kumar coeffs"
+                             "Ce()");
+//  params.addRequiredParam<Real>(
+//      "coef", "The coefficient describing the mismatch between the film and the substrate");
+//  params.addRequiredCoupledVar(
+//      "displacements",
+//      "The displacements appropriate for the simulation geometry and coordinate system");
+  params.addRequiredParam<Real>(
+      "energy_release_rate", "energy release rate or fracture toughness");
+  params.addRequiredParam<Real>(
+      "phase_field_regularization_length", "phase_field_regularization length");
```

Comment:
```suggestion
  params.addParam<Real>(
      "phase_field_regularization_length", "l", "The phase field regularization length");
```
Use a descriptive description here.